x-pack/plugins/security_solution/public/resolver/lib/date.test.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { getFriendlyElapsedTime } from './date';
+
+describe('date', () => {
+  describe('getFriendlyElapsedTime', () => {
+    const second = 1000;
+    const minute = second * 60;
+    const hour = minute * 60;
+    const day = hour * 24;
+    const week = day * 7;
+    const month = day * 30;
+    const year = day * 365;
+
+    const initialTime = new Date('6/1/2020').getTime();
+
+    const oneSecond = new Date(initialTime + 1 * second).getTime();
+    const oneMinute = new Date(initialTime + 1 * minute).getTime();
+    const oneHour = new Date(initialTime + 1 * hour).getTime();
+    const oneDay = new Date(initialTime + 1 * day).getTime();
+    const oneWeek = new Date(initialTime + 1 * week).getTime();
+    const oneMonth = new Date(initialTime + 1 * month).getTime();
+    const oneYear = new Date(initialTime + 1 * year).getTime();
+
+    const almostAMinute = new Date(initialTime + 59.9 * second).getTime();
+    const almostAnHour = new Date(initialTime + 59.9 * minute).getTime();
+    const almostADay = new Date(initialTime + 23.9 * hour).getTime();
+    const almostAWeek = new Date(initialTime + 6.9 * day).getTime();
+    const almostAMonth = new Date(initialTime + 3.9 * week).getTime();
+    const almostAYear = new Date(initialTime + 11.9 * month).getTime();
+    const threeYears = new Date(initialTime + 3 * year).getTime();
+
+    it('should return the correct singular relative time', () => {
+      expect(getFriendlyElapsedTime(initialTime, oneSecond)).toEqual({
+        duration: 1,
+        durationType: 'second',
```

Comment:
:red_circle: probably want the same i18n as you use in `date.ts` in case they test it in different languages?